src/commcare_cloud/ansible/roles/postgresql_base/defaults/main.yml

```diff
@@ -33,3 +33,7 @@ postgresql_slow_log_threshold: 300
 postgresql_random_page_cost: 4.0
 
 allow_direct_postgresql_access: no
+
+postgresql_apt_repo: "deb http://apt.postgresql.org/pub/repos/apt/ {{ ansible_lsb.codename}}-pgdg main"
```

Comment:
For citus repo and binary differs , which can be passed at role dependency
```
postgresql_version: '11'
citus_version: '8.1'
postgresql_apt_repo: "deb https://packagecloud.io/citusdata/community/ubuntu/ {{ ansible_lsb.codename}} main"
postgresql_apt_key_url: "https://repos.citusdata.com/community/gpgkey"
postgresql_binary: "postgresql-{{postgresql_version}}-citus-{{citus_version}}"```

Reply:
the binary that you list here isn't actually a postgres binary - it's a postgres extension that just depends on postgres so if you install it, it will also install postgres.

I'm fine with makint these configurable but I don't think the citus role should change them, rather it should just set the postgresql version, allow the pg role to install postgres, and then it can install the citus extension from the repo you've listed.

Reply:
Okay, that makes sense. I was confused the whole time that it is Postgres binary.